static/src/content/doc/single-node.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Single-Node Batteries Included
+description:
+  Learn how running Batteries Included locally makes development a breeze.
+tags: ['architecture']
+category: getting-started
+draft: false
+---
+
+Modern development cycles should allow users to move between local and
+production workflows as seamlessly as possible. Batteries Included makes this
+possible by providing a consistent toolset that works across environments—from
+your laptop all the way up to enterprise production.
+
+## The Development Journey
+
+With Batteries Included, developers can progress through a natural iterative
+journey:
+
+1. **Local Development**: Start with a lightweight environment locally.
+2. **Staging, Testing**: Deploy to staging with the same configuration.
+3. **Production Deployment**: Confidently launch to production using the same
+   familiar tools.
+
+Using Batteries Included like this eliminates the common frustration of "works
+on my machine" problems, not only reducing onboarding time for new team members
+but also ensuring what works locally works in production as well.
```

Comment:
For now we should drop staging/testing. Then use this as a template for the article.

Local Development should focus on starting, port-forwarding, and deploying batteries. Debugging and other related topics should also be covered. Let's talk about how to get the logs rather than just hinting that we have them. 
Production should focus on how you use the same tools to add more production features. Batteries like monitoring, keycloak, etc., are not likely to be used on local dev but critical for prod.